mearm_positions:
  home:
    joint1: 0.0
    joint2: 0.0
    joint3: 0.0
  forward:
    joint1: 45.0
    joint2: -30.0
    joint3: 30.0
  upward:
    joint1: 0.0
    joint2: 60.0
    joint3: 30.0

